ytmusicapi/ytmusic.py

```diff
@@ -68,10 +70,19 @@ def __init__(self,
         :param location: Optional. Can be used to change the location of the user.
             No location will be set by default. This means it is determined by the server.
             Available languages can be checked in the FAQ.
+        :param oauth_credentials: Optional. Used to specify a different oauth client to be
+            used for authentication flow.
         """
+        self._base_headers = None
```

Comment:
Please group/order the instance variables and add a docstring via `#: this is what it's for`